### PR TITLE
Add Schema based on diagram

### PR DESCRIPTION
**Pull request by AACHOURMOHAMED**
# vit clinic
> Add Schema based on diagram

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+CREATE TABLE invoice_items (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    unit_price DECIMAL,
+    quantity INT,
+    total_price DECIMAL,
+    invoice_id INT,
+    CONSTRAINT ITEMS_FK FOREIGN KEY (invoice_id) REFERENCES invoices(id) ON DELETE CASCADE,
+    treatment_id INT,
+    CONSTRAINT TREATMENT_FK FOREIGN KEY (treatment_id) REFERENCES treatments(id) ON DELETE CASCADE
+)
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql]: - [ ] Kindly add the join table between the `treatments` and `medical_histories` tables as they have a many-to-many relationship, as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Join%20tables%20from%20many%2Dto%2Dmany%20relationships%20might%20not%20appear%20in%20the%20diagram%2C%20but%20you%20still%20need%20them.) requirement.
- [ ] Please add all the foreign keys indices, as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) requirement.

Comment (ShadyShawkat) [inline schema_based_on_diagram.sql]: - [ ] [OPTIONAL] Please update the "Authors" section in your readme to contain all project contributors for your project to look more professional.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### HIGHLIGHTS
- [ ] Nice job adding the indexes for our foreign keys :100: 
- [ ] A join table is created for the treatments and medical_histories table :1st_place_medal: 

### Optional
- [ ] It would be nice if going forward we try and improve our PR, a title and a short summary of project task will do the trick
- [ ] Since Rails follows naming convention over configuration, I would therefore advise we change the column name of our `history_treatment` table from `history_id` to `medical_history_id`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**